roles/confluent.kafka_broker/templates/listener.j2

```diff
@@ -34,8 +37,22 @@ listener.name.{{listener.value.name|lower}}.gssapi.sasl.jaas.config=com.sun.secu
   keyTab="{{kerberos.keytab_dir}}/{{kafka_broker_kerberos_keytab_path | basename}}" \
   principal="{{kafka_broker_kerberos_principal}}";
 {% endif %}
+
```

Comment:
please remove the new lines, although it makes the template file easier to read, it will make server.properties very lengthy